packages/website/docs/DocSearch-v3.mdx

```diff
@@ -0,0 +1,227 @@
+
+### Stable version
+
+With the recent release of the stable version of [Algolia Autocomplete][1], and the huge adoption of DocSearch v3, we will start working on a stable release!
+
+Thanks to all the Alpha testers, and to [all the integrations][5] who already supports it!
+
+## Installation
+
+DocSearch packages are available on the [npm][10] registry.
+
+<Tabs
+  groupId="language"
+  defaultValue="js"
+  values={[
+    { label: 'JavaScript', value: 'js', },
+    { label: 'React', value: 'react', }
+  ]
+}>
+<TabItem value="js">
+
+
+```bash
+yarn add @docsearch/js@alpha
+# or
+npm install @docsearch/js@alpha
+```
+
+If you don’t want to use a package manager, you can use a standalone endpoint:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@docsearch/js@alpha"></script>
+```
+
+</TabItem>
+<TabItem value="react">
+
+
+```bash
+yarn add @docsearch/react@alpha
+# or
+npm install @docsearch/react@alpha
+```
+
+If you don’t want to use a package manager, you can use a standalone endpoint:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@docsearch/react@alpha"></script>
+```
+
+</TabItem>
+
+
+</Tabs>
+
+
+## Get started
+
+<Tabs
+  groupId="language"
+  defaultValue="js"
+  values={[
+    { label: 'JavaScript', value: 'js', },
+    { label: 'React', value: 'react', }
+  ]
+}>
+<TabItem value="js">
+
+
+To get started, you need a [`container`][11] for your DocSearch component to go in. If you don’t have one already, you can insert one into your markup:
+
+```html
+<div id="docsearch"></div>
+```
+
+Then, insert DocSearch into it by calling the [`docsearch`][12] function and providing the container. It can be a [CSS selector][6] or an [Element][7].
+
+Make sure to provide a [`container`][11] (for example, a `div`), not an `input`. DocSearch generates a fully accessible search box for you.
+
+```js app.js
+import docsearch from '@docsearch/js';
+
+import '@docsearch/css';
+
+docsearch({
+  container: '#docsearch',
+  appId: 'YOUR_APP_ID',
+  indexName: 'YOUR_INDEX_NAME',
+  apiKey: 'YOUR_SEARCH_API_KEY',
+});
+```
+
+</TabItem>
+
+
+<TabItem value="react">
+
+
+DocSearch generates a fully accessible search box for you.
+
+```jsx App.js
+import { DocSearch } from '@docsearch/react';
+
+import '@docsearch/css';
+
+function App() {
+  return (
+    <DocSearch
+      appId="YOUR_APP_ID"
+      indexName="YOUR_INDEX_NAME"
+      apiKey="YOUR_SEARCH_API_KEY"
+    />
+  );
+}
+
+export default App;
+```
+
+</TabItem>
+
+
+</Tabs>
+
+
+### Testing
+
+If you're eager to test DocSearch v3 and can't wait to receive your credentails, you can use ours!
+
+```js
+appId: 'BH4D9OD16A';
+apiKey: '25626fae796133dc1e734c6bcaaeac3c';
+indexName: 'docsearch';
+```
+
+### Filtering your search
+
+If your website supports [docsearch meta tags][13] or if you've added [custom variables to your config][14], you'll be able to use the [`facetFilters`][16] option to scope your search results to a [`facet`][15]
```

Comment:
To explain what it can be used for, and it'll be searchable on the documentation when searching for "version" or "language" with DocSearch hehe

```suggestion
If your website supports [DocSearch meta tags][13] or if you've added [custom variables to your config][14], you'll be able to use the [`facetFilters`][16] option to scope your search results to a [`facet`][15]

This is useful to limit the scope of the search to one language or one version.
```